<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeDialog</class>
 <widget class="QDialog" name="DateTimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DateTimeDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDateEdit" name="m_dateEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_editDate"/>
     </item>
     <item row="1" column="0">
      <widget class="QTimeEdit" name="m_timeEdit">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_editTime"/>
     </item>
     <item row="2" column="0">
      <widget class="QDateTimeEdit" name="m_datetimeEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_editDateTime"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnGet">
       <property name="text">
        <string>获取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnSet">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="m_editCalendar">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="m_calendar">
     <property name="selectedDate">
      <date>
       <year>2024</year>
       <month>7</month>
       <day>24</day>
      </date>
     </property>
     <property name="firstDayOfWeek">
      <enum>Qt::Sunday</enum>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::ShortDayNames</enum>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
